Packs/OpsGenie/Integrations/OpsGenieV3/OpsGenieV3.py

```diff
@@ -890,13 +794,35 @@ def remove_tag_incident(client: Client, args: Dict[str, Any]) -> CommandResults:
     if not request_id:
         raise ConnectionError(f"Failed to send request - {data}")
     args['request_id'] = request_id
-    results = client.get_request(args)
-    return CommandResults(
-        outputs_prefix=args.get("output_prefix", "OpsGenie"),
-        outputs=results.get("data"),
-        readable_output=tableToMarkdown("OpsGenie", results.get('data')),
-        raw_response=results
-    )
+    return get_request_command(client, args)
+
+
+def get_request_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    request_type = str(args.get('request_type'))
+    results: Response = client.get_request(args)
+
+    if results.status_code == 404:
+        ScheduledCommand.raise_error_if_not_supported()
+        request_id = args.get('request_id')
+        return CommandResults(
+            raw_response=results,
+            readable_output=None if args.get('polled_once') else f"Waiting for request_id={request_id}",
+            outputs_prefix=args.get("output_prefix", "OpsGenie.Request"),
+            outputs=None if args.get('polled_once') else {"requestId": request_id},
+            scheduled_command=ScheduledCommand(command='opsgenie-get-request',
+                                               next_run_in_seconds=int(
+                                                   args.get('interval_in_seconds', DEFAULT_POLL_INTERVAL)),
+                                               args={**args, 'polled_once': True},
+                                               timeout_in_seconds=int(
+                                                   args.get('timeout_in_seconds', DEFAULT_POLL_TIMEOUT))))
+    else:
+        results_dict = results.json()
+        return CommandResults(
+            outputs_prefix=args.get("output_prefix", f'OpsGenie.{request_type.capitalize()[:-1]}'),
```

Comment:
arent we changing the context path here?

Reply:
yes, depending on what the type is

Reply:
so we're breaking bc?

Reply:
no this is in order to preserve the bc. We had copy pasted code before that would return according to the type